transformers/events/median_drop/transformer_test.go

```diff
@@ -0,0 +1,95 @@
+package median_drop_test
+
+import (
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/core/types"
+	"github.com/lib/pq"
+	"github.com/makerdao/vdb-mcd-transformers/test_config"
+	"github.com/makerdao/vdb-mcd-transformers/transformers/events/median_drop"
+	"github.com/makerdao/vdb-mcd-transformers/transformers/shared"
+	"github.com/makerdao/vdb-mcd-transformers/transformers/shared/constants"
+	"github.com/makerdao/vdb-mcd-transformers/transformers/test_data"
+	"github.com/makerdao/vulcanizedb/libraries/shared/factories/event"
+	"github.com/makerdao/vulcanizedb/pkg/core"
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+)
+
+var _ = Describe("Median drop transformer", func() {
+	var (
+		transformer = median_drop.Transformer{}
+		db          = test_config.NewTestDB(test_config.NewTestNode())
+	)
+
+	BeforeEach(func() {
+		test_config.CleanTestDB(db)
+	})
+
+	It("returns err if log is missing topics", func() {
+		incompleteLog := core.EventLog{}
+		_, err := transformer.ToModels(constants.MedianABI(), []core.EventLog{incompleteLog}, db)
+		Expect(err).To(HaveOccurred())
+	})
+
+	It("returns err if log is missing data", func() {
+		badLog := core.EventLog{
+			Log: types.Log{
+				Topics: []common.Hash{{}, {}, {}, {}},
+			}}
+
+		_, err := transformer.ToModels(constants.MedianABI(), []core.EventLog{badLog}, db)
+		Expect(err).To(HaveOccurred())
+	})
+
+	It("convert a log with 4 accounts to an insertion model", func() {
+		models, err := transformer.ToModels(constants.MedianABI(), []core.EventLog{test_data.MedianDropLogWithFiveAccounts}, db)
+		Expect(err).NotTo(HaveOccurred())
+
+		a1Bytes, aErr := shared.GetLogNoteArgumentAtIndex(2, test_data.MedianDropLogWithFiveAccounts.Log.Data)
```

Comment:
If we're zero-indexing the address name on line 50, then I'd be a fan of zero-indexing `a1Bytes` here as well